src/build/index.js

```diff
@@ -32,4 +32,15 @@ module.exports = (argv) => {
     localDir: path.join(__dirname, '../..'),
     stdio: 'inherit'
   })
+
+  if (argv.bundlesize) {
+    return webpack
+      .then(r => {
+        return execa('bundlesize', ['-f', './dist/index.min.js', '-s', getUserConfig().bundlesize || '100kB'], {
```

Comment:
What about having a `constants.js` for the defaults used in `aegir`? 

We probably have more default constants such as this `100kB` and it would be easier to have those all together. We can start with this one and move the remaining ones later?